kebechet/managers/version/utils.py

```diff
@@ -205,11 +204,29 @@ def _is_merge_event(payload: Dict[str, Any]):
             "This function only works for GitHub payloads at this time."
         )
     if (
-        payload["event"] == "pull_request"
-        and payload["raw_payload"]["payload"]["action"] == "closed"
+            payload["event"] == "pull_request"
+            and payload["raw_payload"]["payload"]["action"] == "closed"
     ):
         return payload["raw_payload"]["payload"]["pull_request"]["merged"]
 
 
 def _pr_id_from_webhook(payload: Dict[str, Any]):
     return payload["raw_payload"]["payload"]["number"]
+
+
+def _is_version_pr(payload: Dict[str, Any]):
+    return "version" in payload["raw_payload"]["payload"]["pull_request"]["title"]
+
+
+def _get_version(payload: Dict[str, Any]):
+    pr_title = payload["raw_payload"]["payload"]["pull_request"]["title"]
+    return re.search(r'version\s*([\d.]+)', pr_title).group(1)
+
+
+def _is_release_pr(payload: Dict[str, Any]):
```

Comment:
can we also check in this function if the pr was opened by kebechet , as it could happen that similar pr is opened in some repo with title `Release of version abc` which is not related to kebechet.
`if "khebhut[bot]" =  payload["raw_payload"]["payload"]["pull_request"]["user"]["login"]`